util/Migrator/DbMigrator.cs

```diff
@@ -130,6 +133,19 @@ public DbMigrator(string connectionString, ILogger<DbMigrator> logger)
         }
 
         var upgrader = builder.Build();
+
+        if (dryRun)
+        {
+            var scriptsToExec = upgrader.GetScriptsToExecute();
+            var stringBuilder = new StringBuilder();
```

Comment:
🎨 Before your loop put a line in here with "Scripts that will be applied:" so it's clear what it's outputting.

Reply:
Done.